src/components/UI/MessagePanel/RenderContent.vue

```diff
@@ -35,6 +35,12 @@ import { mimeToFileType } from '@/lib/util/file'
 import Icon from '@/components/UI/Icon.vue'
 import FileTypeIcon from '@/components/UI/FileTypeIcon.vue'
 import { MarkdownRenderResult } from '@traptitech/traq-markdown-it'
+import {
+  EmbeddingOrUrl,
+  EmbeddingFile
+} from '@traptitech/traq-markdown-it/dist/embeddingExtractor'
```

Comment:
```suggestion
} from '@traptitech/traq-markdown-it'
```
`EmbeddingOrUrl`も`traq-markdown-it`の`index.ts`から`export`するとよさそうです